### ISSUE TITLE
Feature request: confirmation dialog for boost

### ISSUE THREAD

**Issue by hannessssss** [h3poteto/fedistar#1251]
The interaction icons being rather small and close together, it has happened to me occasionally that I accidentally boosted a post, even when I just tried to open hidden media, content warnings etc. Having an option (user configurable?) for Fedistar to ask confirmation before really boosting would be great!

**Comment by tassoman** [comment]
I agree for a configurable behaviour. "Confirm reposts". Then if checkbox is flagged, reboosts are "protected" by a modal window.
Modal window should have two buttons "back" and "Repost". Repost should be default action.
